Resources/doc/controller_events.rst

```diff
@@ -19,12 +19,12 @@ to the response before it is returned.
 Controllers with a form also dispatch an ``INITIALIZE`` event after the entity is
 fetched, but before the form is created.
 
-For instance, this listener will change the redirection after the password
+For instance, this subscriber will change the redirection after the password
 resetting to go to the homepage instead of the profile::
 
-    // src/Acme/UserBundle/EventListener/PasswordResettingListener.php
+    // src/Acme/UserBundle/EventSubscriber/PasswordResettingSubscriber.php
 
-    namespace Acme\UserBundle\EventListener;
+    namespace Acme\UserBundle\EventSubscriber;
```

Comment:
The best practice for the namespace and the class name is still to use ``Listener`` in the name. the fact that the listener is implemented as a subscriber is an implementation detail actually (subscribers are a special way to implement listeners, but they are still listeners)